lib/public/views/Statistics/StatisticsPageModel.js

```diff
@@ -98,7 +99,13 @@ class StatisticsTabbedPanelModel extends TabbedPanelModel {
      */
     constructor() {
         super(Object.values(STATISTICS_PANELS_KEYS));
-        this._remoteDataFetcher = new RemoteDataSource();
+
+        this._remoteDataFetcher = new RemoteDataSource()
+            .feeds(new DataSourceTransfomer((remoteData) =>
+                remoteData.isSuccess()
+                    ? RemoteData.success(remoteData.payload.data)
+                    : remoteData));
```

Comment:
I have some stuff in WebUI I will try to refresh and push, and I'll put in it RemoteData mapping, which should be automated.